### DISCUSSION TITLE
How to hide labels and show title on an Axis?

### DISCUSSION THREAD

**Discussion by Red7M** [patrykandpatrick/vico#445] [Questions] [answered]
I'm using a Chart composable with columnChart() as its chart:

```
Chart(
            chart = columnChart(),
            modifier = Modifier.padding(horizontal = 8.dp),
            model = chartEntryModel,

            bottomAxis = rememberBottomAxis(
                label = textComponent(color = Color.Black, textSize = 12.sp),
                valueFormatter = horizontalAxisValueFormatter
            ),
            endAxis = rememberEndAxis(
                label = textComponent(color = Color.Black, textSize = 12.sp),
                valueFormatter = verticalAxisValueFormatter
            )
        )
```
I want to show only a title in the startAxis, but no label values. I tried: 

```
startAxis = rememberStartAxis(
                title = "Count",
                titleComponent = textComponent(color = Color.Black, textSize = 12.sp),
                label = null
            ), 
```
But it did not work. Help will be appreciated.

**Comment by patrickmichalik** [comment, marked as answer]
Hello! In addition to `label = null`, please use the following:

- `tick = null`
- `itemPlacer = AxisItemPlacer.Vertical.default(maxItemCount = 0)`

That should produce the desired result. We’ll soon be making an improvement that will render `AxisItemPlacer.Vertical.default(maxItemCount = 0)` sufficient for this use case, meaning you’ll be able to skip `label = null` and `tick = null`.

**Comment by patrickmichalik** [comment]
I wanted to follow up and let you know that [Vico 1.13.0 Alpha 9](https://github.com/patrykandpatrick/vico/releases/tag/v1.13.0-alpha.9) introduces the improvement mentioned in my previous comment.
